File: demo/java-demo/src/main/java/com/alibaba/demo/association/SellerService.java
package com.alibaba.demo.association;

/**
 * 被测类，会访问`CookerService`里的方法
 * Class to be tested, which will access methods in TargetService class
 */
public class SellerService {

    private CookerService cookerService = new CookerService();

    public String sellSandwich() {
        return cookerService.prepareSandwich();
    }

    public String sellHamburger() {
        return cookerService.prepareHamburger();
    }
}

File: demo/kotlin-demo/src/main/kotlin/com/alibaba/demo/association/SellerService.kt
package com.alibaba.demo.association

/**
 * 被测类，会访问`CookerService`里的方法
 * Class to be tested, which will access methods in TargetService class
 */
class SellerService {

    private val cookerService = CookerService()

    fun sellSandwich(): String {
        return cookerService.prepareSandwich()
    }

    fun sellHamburger(): String {
        return cookerService.prepareHamburger()
    }
}
